packages/client/src/stores/sidePanel.js

```diff
@@ -32,11 +33,19 @@ export const createSidePanelStore = () => {
     }, 50)
   }
 
+  const setSidepanelState = bool => {
```

Comment:
We should use a more descriptive name here - `setSidepanelState` is very generic and doesn't describe what it's actually doing. I'd recommend something like `setIgnoreClicksOutside` (if you follow my suggestion for how to rename the setting, otherwise adapt appropriately).